tesseract/kinetic/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ros:kinetic-perception
+MAINTAINER Levi Armstrong
+
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
`ARG` is actually better :slightly_smiling_face: 

`ENV` ends up in the resulting image, which causes *all* future invocations of `apt` to be noninteractive, which is not good.